src/session.ts

```diff
@@ -128,6 +139,7 @@ export const loadState = (initial: State): Store => {
           }
 
           try {
+            config.getWalletConnectSigner();
```

Comment:
Can you add a comment explaining why we call this?

Reply:
Done, added more comments in general, to make it clearer for the next time